examples/rails_example/config/initializers/featurehub.rb

```diff
@@ -0,0 +1,16 @@
+
+# configure FeatureHub. Mostly taken from the rails portion of this doc:
+# https://github.com/featurehub-io/featurehub-ruby-sdk/tree/main#2-create-featurehub-config
+Rails.configuration.fh_config = FeatureHub::Sdk::FeatureHubConfig.new(
+    # you may have to change this URL in your app. Assuming you're running the party server locally 
+    # (i.e. what's described here https://www.featurehub.io/) to test out FeatureHub you can find this URL by running
+    # docker ps and finding the container with an image name like 'featurehub/party-server:latest'
+    "http://0.0.0.0:8085/",
+    # You can find this value in the admin console by navigating to the 'API Keys' section of the UI and choosing
+    # between the client and the server API Keys for whatever service account you've set up. Learn more about the
+    # differences between client-side evaluation here:
+    # https://docs.featurehub.io/featurehub/latest/sdks-development.html#_client_side_evaluation
+    # and server-side evaluation here:
+    # https://docs.featurehub.io/featurehub/latest/sdks-development.html#_supporting_server_side_evaluation 
+    ["TODO_CHANGE_ME"],
+).init
```

Comment:
relevant to the review; configure FH

Reply:
yeah, its most likely that people would be using environment variables here? So I would recommend a ENV.fetch using FEATUREHUB_EDGE_URL and FEATUREHUB_CLIENT_API_KEY. Great documentation here! I would never recommend server side evaluation though, so I would stick with just client side evaluation.